[DOC] Starts contribution page (#624)
ME-ICA/tedana: contributions.md

# Contributions

## Contributors
We appreciate all of our contributors!
Each contributor below has made a statement of how they feel they've contributed to `tedana`.

> - Joshua Teves helped manage issues and pull requests for a variety of both administrative and code-specific tasks.

Reviewer: ```suggestion
- Dan Handwerker helps with project management (people wrangling & documentation), led the organization for the 2019 tedana hackathon, provides conceptual feedback on many aspects of the code, contributes to documentation, and, once in a while, even contributes to the code.
- Joshua Teves helps manage issues and pull requests for a variety of both administrative and code-specific tasks.
```

## Funding
Special thanks to the following sources of funding for `tedana`:

> - National Institutes of Mental Health, Section on Functional Imaging Methods for supporting the 2019 `tedana` hackathon.

Reviewer: ```suggestion
- National Institutes of Mental Health, [Section on Functional Imaging Methods](https://fim.nimh.nih.gov) for supporting the 2019 `tedana` hackathon.
```

Author: That's true, we're doing all that work on the website, we should link to it!

> - National Institutes of Health for supporting the 2019 AFNI Code Convergence, where work in the 2019 `tedana` hackathon was continued.

Reviewer: ```suggestion
- National Institutes of Health for supporting the [2019 AFNI Code Convergence](https://codeconvergence.org), where work in the 2019 `tedana` hackathon was continued.
```

# Contributions

## Contributors
We appreciate all of our contributors!
Each contributor below has made a statement of how they feel they've contributed to `tedana`.

> - Dan Handwerker helps with project management (people wrangling & documentation), led the organization for the 2019 tedana hackathon, provides conceptual feedback on many aspects of the code, contributes to documentation, and, once in a while, even contributes to the code.

Reviewer: ```suggestion
- Logan Dowdle helps folks get multi-echo data collection going on their scanners, tries to keep up with the increasing number of multi-echo papers, likes making figures that explain what tedana has done to the data, and occasionally adds a new feature (with lots of help!).
- Dan Handwerker helps with project management (people wrangling & documentation), led the organization for the 2019 tedana hackathon, provides conceptual feedback on many aspects of the code, contributes to documentation, and, once in a while, even contributes to the code.
```

# Contributions

## Contributors
We appreciate all of our contributors!
Each contributor below has made a statement of how they feel they've contributed to `tedana`.

> - Logan Dowdle helps folks get multi-echo data collection going on their scanners, tries to keep up with the increasing number of multi-echo papers, likes making figures that explain what tedana has done to the data, and occasionally adds a new feature (with lots of help!).

Reviewer 2: I think it would be good to either include the person's GitHub username after their full name (e.g., `Logan Dowdle (@dowdlelt)`) or make the name a hyperlink (when desired) to a personal site.

Now that GitHub has personal READMEs, I tend to use my profile for a fairly up-to-date personal description.

Author: I'm happy to link to people's websites if desired, I just need a link if so.

> - Logan Dowdle helps folks get multi-echo data collection going on their scanners, tries to keep up with the increasing number of multi-echo papers, likes making figures that explain what tedana has done to the data, and occasionally adds a new feature (with lots of help!).

Author: With the hyperlink and bolding the names stand out, you can see an example in the rich diff of e99ba95

- Dan Handwerker helps with project management (people wrangling & documentation), led the organization for the 2019 tedana hackathon, provides conceptual feedback on many aspects of the code, contributes to documentation, and, once in a while, even contributes to the code.
- Joshua Teves helps manage issues and pull requests for a variety of both administrative and code-specific tasks.
- Eneko Uruñuela helps with the development of dimensionality reduction and decomposition algorithms in tedana, with Principal Component Analysis to be more specific, and contributed to the development of the interactive reports. 

## Funding

> Special thanks to the following sources of funding for `tedana`:

Author: That's fine. They supported @KirstieJane right?

> Special thanks to the following sources of funding for `tedana`:

Reviewer 3: No, not exactly.. she had a Mozilla fellowship, but that didn't overlap with when she mentored this as a Moz Open Leaders project !

> Special thanks to the following sources of funding for `tedana`:

Author: I'm terribly sorry, do you mind writing a short blurb for what they provided? I just don't know because I wasn't participating at that point.

> Special thanks to the following sources of funding for `tedana`:

Reviewer: More generally: @KirstieJane, We're all adding a sentence or two that self-describes our contributions to tedana. I know your attention is focused elsewhere right now, but I'd like to be able to acknowledge your contributions here. Either make a suggestion to the PR or just put your self-description in a comment & someone else will add it to the text.

- [**Joshua Teves**][joshua-teves] helps manage issues and pull requests for a variety of both administrative and code-specific tasks.
- [**Eneko Uruñuela**][eneko-urunuela] helps with the development of dimensionality reduction and decomposition algorithms in tedana, with Principal Component Analysis to be more specific, and contributed to the development of the interactive reports. 
- [**Maryam Vaziri-Pashkam**][maryam-vaziri-pashkam] helps with the tedana documentation to make it easier to understand for beginners.

## Funding and Operational Support
Special thanks to the following sources of funding and operational support
for `tedana`:
- National Institutes of Mental Health, [Section on Functional Imaging Methods][sfim], for supporting the 2019 `tedana` hackathon.
- National Institutes of Health for supporting the 2019 AFNI Code Convergence, where work in the 2019 `tedana` hackathon was continued.

> - Mozilla Open Leaders for supporting Kirstie Whitaker.

Reviewer 3: ```suggestion
- The Mozilla Open Leaders program for support in developing the tedana community structure as part of [the ME-BIDS project](https://medium.com/read-write-participate/brain-imaging-in-the-open-aac7c17bcf69).
```

# Contributions

## Contributors
We appreciate all of our contributors!
Each contributor below has made a statement of how they feel they've contributed to `tedana`.
- [**Logan Dowdle**][logan-dowdle] helps folks get multi-echo data collection going on their scanners, tries to keep up with the increasing number of multi-echo papers, likes making figures that explain what tedana has done to the data, and occasionally adds a new feature (with lots of help!).
- [**Elizabeth DuPre**][elizabeth-dupre] initiated the tedana project in 2017, building on the ME-ICA codebase. She continued to develop the code and began actively creating the community structure as part of the fifth Mozilla Open Leaders cohort (mentored by Kirstie Whitaker). Since her time as interim BDFL, Elizabeth has been involved in most aspects of the project -- although she currently focuses primarily on improving tedana's integration with the broader neuroimaging ecosystem.
- [**Javier Gonzalez-Castillo**][javier-gonzalez-castillo] contributed to the development of dimensionality reduction and decomposition algorithms in tedana, as well as to the development of the interactive reports.

> - [**Dan Handwerker**][dan-handwerker] helps with project management (people wrangling & documentation), led the organization for the 2019 tedana hackathon, provides conceptual feedback on many aspects of the code, contributes to documentation, and, once in a while, even contributes to the code.
> - [**Joshua Teves**][joshua-teves] helps manage issues and pull requests for a variety of both administrative and code-specific tasks.

Reviewer 2: ```suggestion
- [**Dan Handwerker**][dan-handwerker] helps with project management (people wrangling & documentation), led the organization for the 2019 tedana hackathon, provides conceptual feedback on many aspects of the code, contributes to documentation, and, once in a while, even contributes to the code.
- [**Taylor Salo**][taylor-salo] helps with code-related issues and with engaging new contributors to tedana.
- [**Joshua Teves**][joshua-teves] helps manage issues and pull requests for a variety of both administrative and code-specific tasks.
```

- [**Eneko Uruñuela**][eneko-urunuela] helps with the development of dimensionality reduction and decomposition algorithms in tedana, with Principal Component Analysis to be more specific, and contributed to the development of the interactive reports. 
- [**Maryam Vaziri-Pashkam**][maryam-vaziri-pashkam] helps with the tedana documentation to make it easier to understand for beginners.

## Funding and Operational Support
Special thanks to the following sources of funding and operational support
for `tedana`:
- National Institutes of Mental Health, [Section on Functional Imaging Methods][sfim], for supporting the 2019 `tedana` hackathon.
- National Institutes of Health for supporting the 2019 AFNI Code Convergence, where work in the 2019 `tedana` hackathon was continued.
- The Mozilla Open Leaders program for support in developing the tedana community structure as part of [the ME-BIDS project](https://medium.com/read-write-participate/brain-imaging-in-the-open-aac7c17bcf69).

[logan-dowdle]: <https://github.com/dowdlelt>
[elizabeth-dupre]: <https://github.com/emdupre>
[javier-gonzalez-castillo]: <https://github.com/javiergcas>

> [dan-handwerker]: <https://github.com/handwerkerd>
> [joshua-teves]: <https://github.com/jbteves>

Reviewer 2: ```suggestion
[dan-handwerker]: <https://github.com/handwerkerd>
[taylor-salo]: <https://github.com/tsalo>
[joshua-teves]: <https://github.com/jbteves>
```
